math/mobius.go

```diff
@@ -0,0 +1,38 @@
+// mobius.go
+// description: Returns μ(n)
+// details:
+// For any positive integer n, define μ(n) as the sum of the primitive nth roots of unity.
+// It has values in {−1, 0, 1} depending on the factorization of n into prime factors:
+//   μ(n) = +1 if n is a square-free positive integer with an even number of prime factors.
+//   μ(n) = −1 if n is a square-free positive integer with an odd number of prime factors.
+//   μ(n) = 0 if n has a squared prime factor.
+// wikipedia: https://en.wikipedia.org/wiki/M%C3%B6bius_function
+// author: Akshay Dubey (https://github.com/itsAkshayDubey)
+// see mobius_test.go
+
+package math
+
+import (
+	p "github.com/TheAlgorithms/Go/math/prime"
```

Comment:
Can you keep this `prime` instead of changing to `p`

Reply:
Fixed in [this](https://github.com/TheAlgorithms/Go/pull/532/commits/d784952d98d049ba87023ab3b71230e7bccc4f39) commit